app/assets/stylesheets/forever_style_guide/globals/_effects.scss

```diff
@@ -161,3 +161,37 @@ $color-card-bg: color('white');
     }
   }
 }
+
+// can be added to sections to create slant effect
```

Comment:
to note I had tried to add a working example in the guide quickly in pervious implementation but position absolute was giving me trouble in the existing layout, with the refactor this should not be an issue but in an effort to focus on other related homepage release items I have logged as a to do to circle back to ASAP: https://www.pivotaltracker.com/story/show/150215444